statik/utils.py

```diff
@@ -357,3 +358,27 @@ def find_first_file_with_ext(base_paths, prefix, exts):
 def uncapitalize(s):
     """If the given string begins with a capital letter, it converts it to lowercase."""
     return (s[:1].lower() + s[1:]) if s else ""
+
+
+def find_duplicates_in_array(array):
+    """Runs through the array and returns the elements that contain
+        more than one duplicate
+
+        Args:
+            array: The array to check for duplicates.
+
+        Returns:
+            Array of the elements that are duplicates. Returns empty list if
+            there are no duplicates.
+        """
```

Comment:
the docstring contents and the trailing `"""` should have been under the opening `"""` (I'm doing a quick fix now)